javalib/src/main/scala/java/lang/process/UnixProcessGen2.scala

```diff
@@ -88,7 +88,7 @@ private[process] class UnixProcessHandleGen2(pidFd: CInt)(
     waitWithRepeat()
   }
 
-  private def askZombiesForTheirExitStatus(): Unit = {
+  private def waitAndReapChild(timeout: Option[Ptr[timespec]]): Boolean = try {
     /* This method is simple, but the __long__ explanation it requires
```

Comment:
I believe that the "USE THIS METHOD" paragraph
is now disinformation and should be considered for
removal. 

I think this first paragraph can also be removed. Not as
important as removing the paragraph just described
but worthwhile as long as this block comment is being
edited.